config/settings.yml

```diff
@@ -31,3 +31,5 @@ authorization_workgroup_names:
 file_uploads_root: tmp/uploads
 
 purl_url: https://purl.stanford.edu
+
+terms_of_use: User agrees that, where applicable, content will not be used to identify or to otherwise infringe the privacy or confidentiality rights of individuals. Content distributed via the Stanford Digital Repository may be subject to additional license and use restrictions applied by the depositor.
```

Comment:
Should this go in `config/locales/en.yml`?  This seems more like text rather than a setting.